packages/data-union-solidity/contracts/CloneLib.sol

```diff
@@ -84,7 +84,13 @@ library CloneLib {
         }
         require(proxy != address(0), "error_create");
         if (initData.length != 0) {
-            (bool success, ) = proxy.call(initData);
+            (bool success, bytes memory returndata) = proxy.call(initData);
```

Comment:
return error message from `initialize`? Cool, though this really should be added to the create2 method, and we should maybe even just delete this "old-style create" function and not use it...